src/Microsoft.AspNet.Server.Kestrel/Networking/Libuv.cs

```diff
@@ -2,413 +2,44 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Reflection;
 using System.Runtime.InteropServices;
-using System.Text;
 
 namespace Microsoft.AspNet.Server.Kestrel.Networking
 {
-    public class Libuv
+    public static class Libuv
     {
-        public Libuv()
+        public static bool IsWindows
         {
-            IsWindows = PlatformApis.IsWindows();
-            if (!IsWindows)
+            get
             {
-                IsDarwin = PlatformApis.IsDarwin();
+#if ASPNETCORE50
+                return true;
```

Comment:
Good question.

This code is the same as before, I just took it as is. However, I spent a few thoughts and decided I don't know enough to fix it.